pkg/crc/constants/constants.go

```diff
@@ -41,6 +41,21 @@ const (
 	RegistryURI   = "quay.io/crcont"
 	ClusterDomain = ".crc.testing"
 	AppsDomain    = ".apps-crc.testing"
+
+	GPGPublicKey = `-----BEGIN PGP PUBLIC KEY BLOCK-----
```

Comment:
Might be worth a comment explaining the identity this key corresponds to, link to a public keyserver, ...?

Reply:
@cfergeau yes I will to public keyserver and add link here.